### PR TITLE
Speed based color

### PR DESCRIPTION
**Pull request by jcardus**
https://github.com/traccar/traccar-web/issues/1248

### CODE REVIEW

File: src/map/MapRoutePath.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { useSelector } from 'react-redux';
 import { map } from './core/MapView';
 import { getSpeedColor } from '../common/util/colors';
 
-const MapRoutePath = ({ name, positions, coordinates }) => {
```
Comment (tananaev) [inline src/map/MapRoutePath.js:7]: Why are we removing `coordinates` here?

Comment (jcardus) [reply src/map/MapRoutePath.js:7]: it's not being used anymore

File: src/map/MapRoutePath.js [hunk 2]
```diff
@@ -77,31 +77,32 @@ const MapRoutePath = ({ name, positions, coordinates }) => {
...
+        });
+      }
 
-    map.getSource(id)?.setData({
-      type: 'FeatureCollection',
-      features,
-    });
-  }, [theme, positions, coordinates, reportColor]);
+      map.getSource(id)
+        ?.setData({
```
Comment (tananaev) [inline src/map/MapRoutePath.js:100]: Why is this reformatted?

Comment (jcardus) [reply src/map/MapRoutePath.js]: probably intellij automatically

File: src/map/MapRoutePath.js (hunk 1 above)
Comment (tananaev) [reply src/map/MapRoutePath.js:7]: @jcardus Looks like this is not true. You broke combined report. I appreciate contributions, but please make sure you do proper due diligence before sending pull requests.

**Pull request merged**